Source/RuntimeImageLoader/Public/RuntimeGifReader.h

```diff
@@ -0,0 +1,80 @@
+// Copyright 2023 Peter Leontev. All Rights Reserved.
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "RuntimeGifReader.generated.h"
+
+class UAnimatedTexture2D;
+class FRuntimeGIFLoaderHelper;
+
+USTRUCT()
+struct RUNTIMEIMAGELOADER_API FGifReadResult
+{
+	GENERATED_BODY()
+
+public:
+	FString GifFilename = TEXT("");
+
+	UAnimatedTexture2D* OutTexture = nullptr;
+
+	FString OutError = TEXT("");
+};
+
+UCLASS()
+class URuntimeGifReader : public UObject, public FRunnable
+{
+	GENERATED_BODY()
+
+public:
+	URuntimeGifReader();
+	~URuntimeGifReader();
+	URuntimeGifReader(FVTableHelper& Helper);
```

Comment:
What is this one? You don't need `FVTableHelper& Helper` (basically remove this constructor)

Reply:
Please See UniquePtr usecase as defined in UniquePtr.h. Otherwise we have build error deletion of Unique Ptr